packages/firestore/src/local/persistence.ts

```diff
@@ -75,6 +75,11 @@ export interface Persistence {
    */
   start(): Promise<void>;
 
+  /**
```

Comment:
Nit: I would move the property declarations to the top.

Reply:
Done.